packages/react-native-windows-init/src/Cli.ts

```diff
@@ -34,24 +34,24 @@ const NPM_REGISTRY_URL = validUrl.isUri(npmConfReg)
   : 'http://registry.npmjs.org';
 const npm = new Registry({registry: NPM_REGISTRY_URL});
 
-enum ExitCode {
-  SUCCESS = 0,
-  UNSUPPORTED_VERSION_RN = 3,
-  USER_CANCEL = 4,
-  NO_REACTNATIVE_FOUND = 5,
-  UNKNOWN_ERROR = 6,
-  NO_PACKAGE_JSON = 7,
-  NO_LATEST_RNW = 8,
-  NO_AUTO_MATCHING_RNW = 9,
-  INCOMPATIBLE_OPTIONS = 10,
-  DEVMODE_VERSION_MISMATCH = 11,
-  NO_REACTNATIVE_DEPENDENCIES = 12,
-}
+const ExitCode = {
+  SUCCESS: 0,
+  UNSUPPORTED_VERSION_RN: 3,
+  USER_CANCEL: 4,
+  NO_REACTNATIVE_FOUND: 5,
+  UNKNOWN_ERROR: 6,
+  NO_PACKAGE_JSON: 7,
+  NO_LATEST_RNW: 8,
+  NO_AUTO_MATCHING_RNW: 9,
+  INCOMPATIBLE_OPTIONS: 10,
+  DEVMODE_VERSION_MISMATCH: 11,
+  NO_REACTNATIVE_DEPENDENCIES: 12,
+};
 
 class UserError extends Error {
-  exitCode: ExitCode;
+  exitCode: number;
```

Comment:
Fun fact is that this has the same type safety as before, because TS allows number to enum assignment.